.github/workflows/report.yml

```
@@ -95,77 +148,134 @@ jobs:
       - uses: actions/checkout@v2
       - uses: pinnacles/common-cicd-actions/.github/actions/setup-lang@v0.1.0
         with:
-          use_ruby: ${{ inputs.use_ruby }}
+          use_ruby: 'true'
           ruby_version: ${{ inputs.ruby_version }}
-          use_node: ${{ inputs.use_node }}
-          node_version: ${{ inputs.node_version }}
-          node_package_manager: ${{ inputs.node_package_manager }}
-          use_go: ${{ inputs.use_go }}
-          go_version: ${{ inputs.go_version }}
       - uses: pinnacles/common-cicd-actions/.github/actions/download-rspec-reports@v0.5.3
       - run: |
-          curl -L --out split-test https://github.com/mtsmfm/split-test/releases/download/v1.0.0/split-test-x86_64-unknown-linux-gnu
+          curl -L -o split-test https://github.com/mtsmfm/split-test/releases/download/v1.0.0/split-test-x86_64-unknown-linux-gnu
```

suppress error when I did the curl command like below

```
% curl -L --out split-test https://github.com/mtsmfm/split-test/releases/download/v1.0.0/split-test-x86_64-unknown-linux-gnu
curl: option --out: is ambiguous
curl: try 'curl --help' for more information
```